integration-tests/src/test_loop/tests/bandwidth_scheduler_protocol_upgrade.rs

```diff
@@ -0,0 +1,42 @@
+use std::collections::HashMap;
+
+use near_primitives::types::ShardId;
+use near_primitives::version::{ProtocolFeature, PROTOCOL_VERSION};
+
+use super::protocol_upgrade::test_protocol_upgrade;
+
+#[test]
+fn test_bandwidth_scheduler_protocol_upgrade_no_missing_chunks() {
+    if !ProtocolFeature::BandwidthScheduler.enabled(PROTOCOL_VERSION) {
+        // EpochConfig is unavailable on stable
```

Comment:
Is that comment right? ditto below

Reply:
The test didn't work on stable because of something with `EpochConfig`, so I disabled the test. I could investigate it further, but I don't see a clear benefit.